### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by pulakb**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,7 +1,106 @@
...
+      href: '#',
+      role: 'presentation',
+      tabIndex: 0,
+    },
+  };
+
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--row landing-page__banner">
```
Comment (mattrosno) [inline src/content/LandingPage/LandingPage.js]: It looks like you have this row here twice? `<div className="bx--row landing-page__banner">`

File: src/content/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -1,7 +1,106 @@
...
+              </BreadcrumbItem>
+            </Breadcrumb>
+            <h1 className="landing-page__heading">
+              Design &amp; build with Carbon
+            </h1>
+          </div>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r2">
+        <div className="bx--row landing-page__r2">
```
Comment (mattrosno) [inline src/content/LandingPage/LandingPage.js]: This also looks like you have an unnecessary, duplicated `<div>` here.

File: src/content/LandingPage/LandingPage.js (hunk 1 above)
Comment (pulakb) [reply src/content/LandingPage/LandingPage.js]: Duplication removed!

File: src/content/LandingPage/LandingPage.js (hunk 2 above)
Comment (pulakb) [reply src/content/LandingPage/LandingPage.js]: Duplication removed!

**Pull request closed without merging**